docs/.eleventy.js

```diff
@@ -78,11 +79,20 @@ module.exports = function(eleventyConfig) {
   });
 
   // Add syntax highlighting
-  eleventyConfig.addPlugin(syntaxHighlight);
+  eleventyConfig.addPlugin(syntaxHighlight, { className: "s-code-block" });
 
   // Add markdown shortcode
   eleventyConfig.addPlugin(markdownShortcode, {
-    html: true
+    html: true,
+    highlight: function (str, lang) {
+      if (lang && hljs.getLanguage(lang)) {
+        return '<pre class="language-' + lang + ' s-code-block"><code class="language-' + lang + ' s-code-block">' +
```

Comment:
This matches the output from our custom `eleventy-highlight` plugin, but it's probably a little weird.

Reply:
I'm not 100% sure what this plugin actually does (embed markdown inside other template types?), but our highlight plugin should support markdown templates already. See the docs for the [official 11ty plugin](https://www.11ty.dev/docs/plugins/syntaxhighlight/#markdown-highlighter) that I ripped off of for examples

Reply:
Also, this looks *suspiciously* like [markdown-it](https://github.com/markdown-it/markdown-it#syntax-highlighting) 😉. The "official" approach is very similar to the one you took here.

Reply:
I tried this out. We don't want to highlight the syntax of Markdown here. Instead, we want to render all the markdown as HTML and shove it into our documentation. Like so:

```
<div class="s-prose">
    {% markdown '/_includes/markdown-sandbox.md' %}
</div>
```

I'm sure there's an opportunity to clean this up, but in the nearterm, this matches if I'd just written all that HTML inside of the `s-prose` content, including the syntax highlighting.

Reply:
Ah, I get ya. It isn't a markdown 11ty template, it's straight up a raw markdown file getting read and rendered. I don't feel strongly about how we're doing this, just figured I'd mention it. If we migrate that file to an 11ty markdown template in the future we could take advantage of the built in highlighter, but until then this is more than good enough.